### DISCUSSION TITLE
Q3S1 (July to Mid-Aug) Roadmap Planning

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#4405] [Ideas]
![yuouarehere](https://user-images.githubusercontent.com/158833/123798602-3f863a00-d8df-11eb-9df8-3e748616e3db.png)

OMG another six weeks have passed and it's time to collect our thoughts and plan the remainder of 2021. Having followed an asynchronous approach the last [two](https://github.com/opencollective/opencollective/discussions/4273) [sprints](https://github.com/opencollective/opencollective/discussions/4161) we thought we should continue to evolve the format. 

## Our priorities

> Our priorities do not change, but our framing does. We continue to focus on our core strengths but as a product team, we shall consolidate on our existing success.

2021 has been about deepening our core strengths. It has been great to focus on what makes Open Collective unique, but we have continued to prioritize new features over honing existing experiences that can still limit and frustrate us. Simultaneously 2021 has seen the Open Collective family continue to succeed, particularly in first-party hosts where we're able to leverage our flexibility and our experience to accelerate growth. Experience that we should build upon and share. 

In order to free the team to develop first and third-party hosts, and to focus on the things that are often lost in the heat of feature development, we felt that consolidating our success should be the focus for the remainder of the year. That we should focus on fundamentals like the ledger, completing unfinished projects and, addressing issues that cause ourselves and our user's significant pains and limit our growth.

So what do we _not_ prioritize? We should try not to focus on the needs of smaller hosts, self-hosted collectives, and significant, new feature development. This does not mean that we ignore these needs, and we should continue to listen to our users and understand their concerns, but we should seek to understand their impact on the whole rather than the select few. 

## How do I contribute?

In previous sprints we have focussed on _projects_ as the level of granularity that we propose projects. Due to the nature of the priorities above you may instead have a list of smaller issues that you feel are particularly impactful and aligned with our priorities. 

That's fine. 

If you're able to group issues and problems into 'themes' this will be helpful as we plan the next six weeks. Otherwise, we (by which I mean @benjam) will work to organize issues raised along broad themes so that we don't spend the next six weeks to context-switching and discussing minutiae. 

#### Proposing issues
Create a top-level comment on this discussion, link to any issues that already exist in this repo (a quick search is good enough) but don't worry if you don't have the time or can't find anything. Content is more important than linking.

#### Discussion
Feel free to lend thoughts, opionions, data and ask questions in threads from each of the top-level comments (which should contain project).

#### Voting
Feel free to 👍 any project ideas that you feel would be a good fit for this sprint. Try to limit yourself to three votes for the sprint to keep things focussed.

#### Synchronous discussion
We'll arrange a synchronous discussion for everyone Thursday 1st to talk through the top pitches for the quarter and to create the space for us to decide together based on the outcome of this discussion.

#### Kickoffs and further discussion
Depedning on the level of shared understanding between product, design and engineering we may not need a kick-off for each individual issue that that issue contains the necessary detail. A quick 1:1 1:many discussion to scope the issue(s) within a theme may get us through.

**Comment by BenJam** [comment]
## Virtual Cards: Redux 

I've been gathering feedback on virtual cards over at https://github.com/opencollective/opencollective/issues/4312#issuecomment-869672670 within that list I think the following would be great for our own health:

- stop collectives that have not been accepted requesting a card
- stop collectives without a balance requesting a card
- show the number of cards that are already allocated when allocating a new card
- allow collectives to add a purpose and limit to card requests
- pausing all cards once the balance goes to zero

We might also want to issue a reminder then deactivate unused cards after 30/60 days (respectively).

**Reply by alinamanko** [reply]
Linking to the proposal on events - I wonder if there is a way to see how we could create an easy link of cards to be used for a specific event/project rather than a collective? This is a specific situation we encountered [with Mautic](https://github.com/opencollective/opencollective/issues/4402) but I do realize this may require actually building this capacity and I would consider maybe raising it again in the next round as it may be more complex, but just putting it here for those of you more equipped than me to evaluate this idea :)

**Reply by alanna** [reply]
Do we have any data about uptake of virtual cards? How might we measure their success? Are Collectives with virtual cards creating more expenses or spending more $ than those without? The hypothesis was people were skipping expensing small things and recurring subscriptions because it's a pain to upload expenses, and virtual cards would help with this. Has it?

**Reply by alinamanko** [reply]
I am happy to look into this more closely @alanna and report back. Additional impact to having virtual cards is it reduces the impact on operations when processing expenses (but this could also be measured as a tradeoff to time of processing new requests received)

**Reply by kaylarep** [reply]
not sure if this is resolved, but if edits to a card's max card budget are made in Privacy, would be good to have those updates auto-sync into the platform

**Reply by BenJam** [reply]
> Do we have any data about uptake of virtual cards?

I added to Leo's dashboard with some metrics over at https://opencollective-metabase.herokuapp.com/dashboard/55

>The hypothesis was people were skipping expensing small things and recurring subscriptions because it's a pain to upload expenses, and virtual cards would help with this. Has it?

This is precisely the kind of critique I want to bring to all our work <3. certainly the first litmus test of 'will people use it' has been passed, with [just under 1% of expenses](https://opencollective-metabase.herokuapp.com/question/354) in the Open Source Collective host using virtual cards. It's small, but it's a start. I think we should track this figure specifically, aiming to get to 5% of expenses in the next 6 weeks.

**Reply by BenJam** [reply]
> I wonder if there is a way to see how we could create an easy link of cards to be used for a specific event/project rather than a collective

+1 this is so obvious but I was so concerned with the mechanics of card limits that I missed adding the purpose with the request. We just have a 'notes' section and I think we could either lead with guidance here (I can already do this by adjusting the policy) or we could explicitly request the purpose of the card.

I've also noted that some requests are coming through without a specific administrator name in the email. I think this might be a bug in the email template so I'll create an issue for it.

**Comment by piamancini** [comment]
Adding a little loom here about the thinking behind this proposal! https://www.loom.com/share/21bc1e79245b4b819ab3f4476b28a12b

**Reply by Betree** [reply]
:100: with this. We have many things to consolidate, and having some space for this would greatly impact quality.

As part of this, I would like us to dedicate some time to security and performance during the next sprints. We have clear, actionable items to improve things on these topics. That would make us more confident in our capacity to scale.

**Reply by SudharakaP** [reply]
Agree with this wholeheartedly as well. 😄

**Reply by kewitz** [reply]
Yes. I think there are a few things that were left behind or are still a bit "beta" around our code that we should consolidate.
Make new implementations default, delete some deprecated code, improve tooling and fixing bugs. 💯

**Reply by Memo-Es** [reply]
On the design side, this perspective for the next 6 months allows us to focus on the marketing side of Open Collective, which is quite scarce regarding the love we could add to how our users perceive our brand and make it easy for them to share their experience of using the platform and its benefits.

**Reply by kaylarep** [reply]
> On the design side, this perspective for the next 6 months allows us to focus on the marketing side of Open Collective, which is quite scarce regarding the love we could add to how our users perceive our brand and make it easy for them to share their experience of using the platform and its benefits.

also, could be cool to develop some sort of feedback system, to hear how the users feel about the accessibility of all our features

**Comment by SudharakaP** [comment]
## Remove Express Custom Server and Move to NextJS Default Server

I do have a small project suggestion maybe for the next six months or later, depending on our capacities + if the engineering team thinks it's an idea worth spending time on. 😄 

Basically I think that it would be nice to remove our [dependency on the custom express server](https://nextjs.org/docs/advanced-features/custom-server) and move completely to the default `Next.js` server. This will add a lot of performance enhancements overall to the web application and would be a step towards making sure the application is ready to scale further in my opinion. Thus I would propose this as a potential project. 😄 

I will open a dedicated project issue when I do some more investigations on this matter. But let me know your thoughts. 😄 

@opencollective/core-developers

**Reply by Betree** [reply]
Could be a good scoping project for this sprint, to at least list what needs to be migrated and estimate the cost

**Reply by znarf** [reply]
Agree with Ben, I don't want us to tackle such project if there is no extreme clarity on what needs to be done.

I'm skeptical on "a lot of performance enhancements overall" and would like to see facts. I would also like to know about the honest downsides.

**Reply by SudharakaP** [reply]
@Betree @znarf : I am also not 100% sure on whether we should or shouldn't do this. I don't think honestly that not migrating to express will pull us back, but rather it might help improve certain things. I did some investigation today and these are some **benefits** I could find from the NextJS docs. 😄 

**Benefits**

- Server-less Functions: Pages that use SSR and API routes will automatically become serverless functions; 

>Pages that use Server-Side Rendering and API routes will automatically become isolated Serverless Functions. This allows page rendering and API requests to scale infinitely.

- [Automatic Static Optimization](https://nextjs.org/docs/advanced-features/automatic-static-optimization): NextJS claims that Automatic Static Optimization will make page loading ultra-fast. 

>One of the main benefits of this feature is that optimized pages require no server-side computation, and can be instantly streamed to the end-user from multiple CDN locations. The result is an ultra fast loading experience for your users. 

I feel that if the docs are true to their word, the second point might be interesting to consider. The docs also mentions that the determination whether a page should be statically optimized is based on whether `getServerSideProps` and `getInitialProps` are absent in the page. I did a count and out of the ~600 components around ~50 components have these functions. And as such, we might be able to leverage some gains in page loading. 

**DownSides**

- This might involve some significant effort I feel; especially since we are using lot of different express middleware.

- We might be stuck with Vercel. Note that those two benefits seems only to be guaranteed if we deploy on Vercel. There are some problems if we use something other than Vercel. For example note the discussion; https://github.com/vercel/next.js/discussions/17563. 

Especially given the second point above, I am reconsidering my proposal and kinda hesitant on opening this as a project. 😄 My thinking is although we are missing out on some optimizations that Vercel provides it might not be worth the effort as we will be losing some flexibility being stuck with Vercel. Anyways, let me know your thoughts. 😄

**Reply by Betree** [reply]
> This might involve some significant effort I feel; especially since we are using lot of different express middleware.

We know that already, the goal of this project is precisely to describe what we need to do and estimate the cost. As mentioned above, we need a scoping project before any implementation can be kicked off.

> We might be stuck with Vercel.

Why? Isn't the issue you shared about problems when using a custom Express server (which this project is about getting rid of) rather than Vercel itself?

> Especially given the second point above, I am reconsidering my proposal and kinda hesitant on opening this as a project

It hasn't been prioritized for this sprint, so there's no rush. I think we're still interested in the scoping part, moving to Vercel (which is, in fact, getting rid of our custom Express server to switch to NextJS's default one).

It would be great to move this discussion to a dedicated issue.

**Reply by SudharakaP** [reply]
@Betree : Thanks, let me create a dedicated issue for this one with some more investigations soon. 😄 

>Why? Isn't the issue you shared about problems when using a custom Express server (which this project is about getting rid of) rather than Vercel itself?

Yes correct, my initial thinking was that the custom server has some limitations in terms of performance. However when I investigated more yesterday my thoughts were that when we move to Vercel we get rid of those maybe (specifically adding serverless-functions and also better support for static optimizations), but we are dependent on Vercel. So for example just getting rid of custom server but not moving to Vercel will not get the full benefits of the static optimizations for example; as per the linked issue I've highlighted above; https://github.com/vercel/next.js/discussions/17563. So this is what I meant by getting stuck with Vercel. In essence we are inclined to use Vercel which I thought wasn't ideal. 😄 

But in any case let me create an issue for that and we'll discuss this more there. 😄

**Reply by SudharakaP** [reply]
> It would be great to move this discussion to a dedicated issue.

I've created a dedicated project issue at, https://github.com/opencollective/opencollective/issues/4439. We can discuss further there. 😄

**Comment by Betree** [comment]
# Ledger follow-up 1: fix Metabase queries

As part of what was mentioned in https://github.com/opencollective/opencollective/issues/4124#issuecomment-870414663, there are a few things we need to iterate on following the ledger refactors that we've done during the previous sprint. This part is **essential** as we'll break other flows if we don't iterate quickly. What's to do:

- Review accountant reports in Metabase
- Review revenue metrics in Metabase

## Complexity
Difficult to estimate. Between 1 and 10 days?

**Reply by znarf** [reply]
We can do our best effort to update current metabase queries, this is something that I have started and should not be too complicated.

If the queries are important enough, I think they should be moved to the API project and exposed directly from there.

The main reasons are that we need:
- versioning: it will be cleaner to have an history of the queries, know what changed and when, being able to update them directly in the PR when we break something
- mutalisation: there is a lot of copy/paste currently in different metabase queries and it would be a cleaner approach to mutualise common parts

We could then expose the results using GraphQL. Visualizations could be done with a small graphing library integrated in the frontend or in a completely separate project.

**Reply by BenJam** [reply]
DONE!

**Comment by Betree** [comment]
# Ledger follow-up 2: Improve CSV reports

As part of what was mentioned in #4124 (comment), we think it's a good time to review the CSV reports that haven't really been refactored in years, both to clean the code and to surface the new information that we have. What we want to achieve
- Surface host fee share
- Merge the code to generate CSV between frontend (transactions list) and API (emails)
- Show transactions groups
- Update descriptions for host fees (+ shares) - eg. "Host Fee share for transactions group #xxxxx-yyyy-zzzz-aaaa"

## Complexity

2-8 days

**Edit**: Created a project issue here: https://github.com/opencollective/opencollective/issues/4426. Also updated the roadmap.

**Reply by BenJam** [reply]
100% here, conversations with NumFocus have highlighted that we need to add an on demand export of transactions from host dashboards.

**Reply by alinamanko** [reply]
I think that is a really great suggestion @Betree! I specifically would be interested in the code merge as having access to contributors' contact details is a common question we receive from collective admins and it would be really good if we could make their lives a little easier in this space.

**Reply by alanna** [reply]
Related #3983 and possibly others linked from #4396

**Reply by Betree** [reply]
https://github.com/opencollective/opencollective/issues/4415 can also be considered for this project

**Comment by Betree** [comment]
# PayPal subscriptions

I mainly want to tackle [PayPal synchronization issues #4368](https://github.com/opencollective/opencollective/issues/4368) which is an absolute priority for me as it creates inconsistencies in our ledger. 

As bonuses, we can also add:
- https://github.com/opencollective/opencollective/issues/4225 (almost done already)

## Complexity
3-7 days

**Comment by Betree** [comment]
# Performance improvements / caching

This project might be a bit difficult to achieve this sprint, but I think that a little focus on performance could greatly improve our loading times and fix some page crashes happening on large hosts like OSC (Timeout - Heroku Application Error). It's also an opportunity to look at our caching strategy, which we know from user feedback is causing some issues (eg. "I've edited X, but I don't see it on my collective page").

The following issues would be a good start:
- https://github.com/opencollective/opencollective/issues/3788 
- https://github.com/opencollective/opencollective/issues/3996

## Complexity

3-8 days

**Reply by piamancini** [reply]
yes! performance is sometimes quite slow depending on the page you are loading. And +1 to making cache more amicable.

**Reply by BenJam** [reply]
Do we have a good understanding of page load time for P50, P95 and P99? it would be good to gauge the impact of this work on overall performance.

**Reply by alinamanko** [reply]
Another point to consider with slow performance or crashing pages is it creates a risk of transactions recorded incorrectly. We had more instances than what we would like where users made a contribution but due to a problem of page loading, a situation occurred when their card was charged twice and this is not a good experience even if we are quick to rectify the double payments. It also creates some risk when processing payments.

**Comment by piamancini** [comment]
Question: How are we handling the crypto / open stocks in this framework? We are half way there already, is there an easy path for us to have a tier that links to the external checkouts? 

My thinking is that since we are not integrating with the services / it's a link to a checkout it's worth it for us to close the loop in these 
two. 

I would like to launch after the summer at the latest. Thoughts?

**Reply by BenJam** [reply]
tbh I have zero little where we are with the design/development of this other than having selected a provider... 

If the work is mostly done, then I think we should complete it. If not perhaps we should see where we are with other priorities.

**Reply by piamancini** [reply]
I think that as long as we have a place to link to the checkouts we are good to test this.

**Reply by alanna** [reply]
How about we scope it to an MVP release for now? We should at least spec out what an MVP would look like in more detail.

**Reply by alanna** [reply]
I think crypto/stock donations are *really* exciting, especially for OSC. To the point that OSC might want to consider spending some of its own budget to get them over the line if OC's capacity is maxed out. cc @BenJam

**Reply by piamancini** [reply]
+1

**Comment by BenJam** [comment]
## Events 

Having just looked through the data, it _does_ appear that events are starting to come back. In total events account for around $500k of funding for collectives and we've seen some movement on ticket sales https://opencollective-metabase.herokuapp.com/dashboard/58

I proposed to review the issues stacked up for Events should we wish to pick up development @alanna did an amazing job already over in https://github.com/opencollective/opencollective/projects/29 but for me a few stand out:

- fix sending funds between events and parent collectives showing as a contribution https://github.com/opencollective/opencollective/issues/4382 (which is probably true for projects too)
- making past events clearer https://github.com/opencollective/opencollective/issues/3262
- prompt admins to move budgets from events to parent collectives once over (as per @alanna's request)
- Add 'attendee' status to collectives that are events instead of 'backer' to enable better tracking
- Refund/cancel events as an attendee

**Reply by alanna** [reply]
I feel like Events is the poster child for "half-finished feature we never consolidated properly" so to me, it really fits in the prioritisation frame. When @piamancini said "make our users happy" I immediately thought of events. It also has the potential to bring us new users and is the first experience of the platform for many.

I am currently trying to use Humanitix for our Fiscal Host calls and it pretty much sucks. There are no good tools out there for that kind of recurring event. I'd love to be able to use our own platform instead.

**Reply by BenJam** [reply]
> There are no good tools out there for that kind of recurring event

This is interesting. Are events primarily a means for communities to organize or for communities to fundraise?

**Reply by alanna** [reply]
> Are events primarily a means for communities to organize or for communities to fundraise?

Yes! But if I had to pick one, I'd probably say community organizing, since many events run at cost. But that community building also has impacts for fundraising.

**Comment by alanna** [comment]
Totally agree with the framing here! I will come back soon and put some ideas forward.

> a synchronous discussion for everyone

Unfortunately these words can't all be in the same sentence, just like we can't all be in the same meeting 😩

**Reply by BenJam** [reply]
true, sorry.

**Comment by alanna** [comment]
# Integrate with accounting tools and practices [#4396] 

This is related to the two ledger ones @Betree posted above, but I'm also posting this project because it's a different perspective on some of the same issues. I think to really get the ledger stuff right we need to include both technical and accounting expertise in the mix. So maybe we can think about how best to frame these related projects. 

This project represents a key pain point for OC inc (see issues with the revenue dashboard), OSC and OCF (issues working with the accountants), and potential new established third part hosts (every single one asks how they can integrate OC with quickbooks or xero).

**Reply by alanna** [reply]
I think #4360 is actually critical to this but it hasn't gotten much attention. We're about to scale up use of Projects on the platform and we really need to ensure we're handling them properly in terms of how the ledger and accounting reporting works.

**Reply by BenJam** [reply]
With respect to projects, this may well be a feature rather than a bug. For instance if I run a collective that has multiple projects (say, the DI Fund) I may want and need to distinguish spend on the administration of the fund itself from the funds that are allocated to projects. So I think the best thing is to have both.

**Reply by alanna** [reply]
Running transactions through the parent then to the project will achieve both, but not doing so causes all kinds of problems.

**Comment by alanna** [comment]
# Small Improvements for Hosts

I don't know if these should be collated into a project issue or what, but I'll just put them here for now. These are things related to hosts that we've left half-finished or are painful papercuts. I *think* all of these are not hugely complex and should be considered as consolidating/finishing/polishing (although some might be more complex than I hope?).

* #3330 
* #3193
* #2757
* #4024 
* #3901 
* #4056 
* #4251 
* #4144 
* #4009

**Reply by BenJam** [reply]
@alinamanko @alanna is there anything above that you would say is causing more pain than the others?

**Reply by alanna** [reply]
I'd vote for #3330, #4024, and #4009 - but some of the others are small and simple and could be easy wins so they should be considered too.

**Comment by alanna** [comment]
What is our plan for stuff that's already been designed and we've started working on? For example #3624 ? Are things like this automatically in because they are half done?

**Reply by BenJam** [reply]
++ for things that have already been design @Memo-Es @aminakazi do y'all have a backlog of designs that are completed? maybe we should create one?

**Reply by BenJam** [reply]
@Memo-Es to the rescue!

**Reply by znarf** [reply]
They should always be considered but they can not always be prioritized.

They should not have a fast track and we should tackle them only if there is a momentum and general agreement.

**Comment by alanna** [comment]
# Privacy on a Transparent Platform

Where are we at with #4214?

To me, the most important related issues are #3689 (which is super important for accounting processes as well) and #3422.

**Reply by BenJam** [reply]
> Where are we at with #4214?

I'm getting there this morning :)

**Reply by BenJam** [reply]
okay so I'm with you on #3689: operating under a single pseudonym does come with some risk, but it's. ahigher risk allowing incognito usage from an abuse on others point of view. I think we might be able to work #3422 to use a pseudonym and show the real name at the point we're paying expenses.

**Reply by alanna** [reply]
Incognito payouts would definitely still need to show the real payment info to the admins.

**Comment by kaylarep** [comment]
# Search experience redesign

bringing this one back in... our search bar could really really use improvement. the fact that everything has to be spelled perfectly/ there aren't enough ways to find the hosts/collectives is really limiting  https://github.com/opencollective/opencollective/issues/3892

**Reply by BenJam** [reply]
++ I think @Memo-Es has been working some improvements to search this last six weeks, we need to catch up to design here :)

**Reply by Memo-Es** [reply]
I went ahead and changed the title. It is a project I consider to be part feature and part marketing so it also makes sense with our intention to tackle marketing-focused projects in design.
We just scoped it and its ready to be designed https://github.com/opencollective/opencollective/issues/4295

**Reply by kaylarep** [reply]
awesome :) thanks yall! +1 for part feature/part design

**Comment by alanna** [comment]
# Smash the bugs 🪲

We currently have [78 open bug issues](https://github.com/opencollective/opencollective/issues?q=is%3Aopen+is%3Aissue+label%3Abug). If we're focusing on fixes and improvements, let's devote some capacity to bringing that number down.

**Reply by BenJam** [reply]
++ though not all bugs are equal: https://github.com/opencollective/opencollective/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Aimportant

**Comment by Memo-Es** [comment]
# Implement projects that are ready for production

There are a handful of projects ready to be implemented; I'm posting each as a comment in this thread:

**Reply by Memo-Es** [reply]
## Admin dashboard update https://github.com/opencollective/opencollective/issues/3880

This project was born out of the idea of merging the host dashboard and the settings for hosts, the final design contemplates the migration of all user types ( Hosts, collectives, organizations and users) to have an admin dashboard.

**Reply by Memo-Es** [reply]
## Fiscal hosts settings https://github.com/opencollective/opencollective/issues/2665

Related to https://github.com/opencollective/opencollective/discussions/4405#discussioncomment-949700 we could dedicate some time to update the settings for hosts now that we will have the updated look of the admin dashboard

**Reply by Memo-Es** [reply]
## Request a grant https://github.com/opencollective/opencollective/issues/3778

Improvements to the design of the expense flow to adapt to grants requests.

**Reply by Memo-Es** [reply]
## Budget page design for collectives (Transparency page) https://github.com/opencollective/opencollective/issues/3998

The design of a dedicated page to display a collective's transparent budget, it's also good to start using these graphs to test them before implementing them in the hosts' admin dashboard.

**Reply by Memo-Es** [reply]
## Onboarding improvements
* https://github.com/opencollective/opencollective/issues/4256
* https://github.com/opencollective/opencollective/issues/3758

**Reply by Memo-Es** [reply]
## Join a host experience https://github.com/opencollective/opencollective/issues/4240

Design of the page to find and join a host when you're a collective, in a standalone version and embedded in the settings for collectives.

**Reply by Memo-Es** [reply]
## Improvements to the settings

Updates to some pages in the settings were requested in the last few months:
* https://github.com/opencollective/opencollective/issues/3357
* https://github.com/opencollective/opencollective/issues/3438
* https://github.com/opencollective/opencollective/issues/3624

**Reply by Memo-Es** [reply]
## Contribution flow update https://github.com/opencollective/opencollective/issues/4299

We are about to finish the production files of an update to the contribution flow (Platform tips UI, layout, and tier cards)

** *Implementation issue pending* **

**Reply by BenJam** [reply]
oh wow, this is a bucket list. Thanks for pulling this together @Memo-Es.

**Reply by alanna** [reply]
Please hold of on #3778 - OCF is right in the middle of a grants policy update and we'll have a lot of input once we're clearer on how it needs to work from a compliance standpoint.

If we consider all of the above in progress and try to implement all of them, I think it will stray into the territory of "shiny new features" instead of "fixes and consolidation". At least for some of the more ambitious ones. Of course I want all of them to be implemented! But I'm concerned about capacity and focus.

**Reply by BenJam** [reply]
:wave: moving a conversation over from slack/zoom (because hello @opencollective/apac) : 

We still have a lot of design here (in this thread) ready to roll from a design point of view, and we have some time from engineering to dedicate to implementing [next sprint](https://github.com/opencollective/opencollective/discussions/4545) so we'll continue the conversation here then pull the top 2-3 'projects' into this sprint for prioritisation.

**Reply by BenJam** [reply]
Personally, I think that we should work on the host dashboard https://github.com/opencollective/opencollective/issues/3880 and https://github.com/opencollective/opencollective/issues/4300  followed closely by the disambiguation of the signup flow for collectives and hosts https://github.com/opencollective/opencollective/issues/4256 https://github.com/opencollective/opencollective/issues/3758

**Comment by BenJam** [comment]
## Funds
So there are a couple of issues that I've found using funds that I've documented over at https://github.com/opencollective/opencollective/issues/4388 and https://github.com/opencollective/opencollective/issues/4389 \(https://github.com/opencollective/opencollective/issues/3981 is also relevant) but I also have another issue that I mentioned in slack with respect to when the host fees are taken.

That is if I ad $500 to a fund, so that I can back vuejs for the gold sponsorship level, I will actually $450 to give once fees are taken... which means I can't back vuejs at the (500) Gold level...

This is actively working against the purpose of funds for me, in so much as often (still) companies looking to support open source projects are doing so from budgets that originate from marketing departments and that the halo effect of having a certain sponsorship level a big part of why the organisation is supporting.

**Reply by alanna** [reply]
This is also blocking some orgs who currently use prepaid budgets from switching to Funds, because they want credit for giving $x, not $x minus fees, on the recipient's page. When someone contributes $100 directly, it shows as they gave $100 on the leaderboard, not $90, even if the Collective actually got $90 after fees. I think this is a problem because supporting both prepaid budgets and Funds is technical/support debt.

**Reply by BenJam** [reply]
> I think this is a problem because supporting both prepaid budgets and Funds is technical/support debt.

100% I am probably going to have to hand hold some corporate though it :(

**Comment by Memo-Es** [comment]
## Marketing update (Design)

We can tackle some basic points to cover during our marketing-focused semester

* Make sure all of our marketing pages have the updated style
* Navbar design update
* Presentation and visual assets for the team
* Scope the new homepage project
* Assets library (OC, OSC, OCF, and other initiatives)
* Media resources (Explanatory clips, like the video on homepage)

**Comment by znarf** [comment]
### Contact Details Sharing opt-in

For privacy reasons (GDPR, etc ...), we need to start asking people to consent sharing their contact details (email) with the Collectives they support. It doesn't seem legally acceptable to share it with them by default anymore.

Alternative could be to update Privacy Policy and Terms of Service, but this would need to be backed by Legal.

**Reply by BenJam** [reply]
https://github.com/opencollective/opencollective/issues/4537

**Comment by BenJam** [comment]
## Defining the relationship between collectives, related collectives tiers, projects, events
Picking up https://github.com/opencollective/opencollective/issues/4281 as I think that we might be able to get through it primarily with documentation.

> We've talked about how some of our features have evolved to occupy overlapping space: Projects, Events, Connected Collectives, and Tiers are all different ways to have mini-Collectives or mini-budgets inside or connected to the main one. We've continued to build on them all without stepping back to think more deeply and avoid feature bloat. Intuition says there may be a more elegant approach if we take the time to fully think it through.

**Comment by BenJam** [comment]
## Projects
Adding here for my own benefit because I don't think projects has had a proper release yet, but since we've completed #4213 we should be ready to go.

**Comment by kewitz** [comment]
# Consolidate Expenses Batching UX (PayPal Payouts and Wise)

Right now scheduled PayPal Payouts expenses are automatically paid while Wise expenses are not.
I would like to make this consistent and requiring users to review and pay for a batch manually.
There are also a few tweaks to this like creating Wise recipients on expense creation, moving the burden of invalid recipients error from the host admin to the user submitting the expense.

**Reply by BenJam** [reply]
👍  going to pause on this for a little while and come back to it if we get feedback that have two processes is confusing or painful.
